### PR TITLE
Refactor MTE-3388 Report test results to non-sandbox slack channel

### PR DESCRIPTION
**Pull request by clarmso**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-3388)

## :bulb: Description

This PR adds the webhook for the non-sandbox channel. See the repository's settings for the token. Here are the workflows that would no longer report to the sandbox channel:

* https://github.com/mozilla-mobile/firefox-ios/actions/runs/10911567226
* https://github.com/mozilla-mobile/firefox-ios/actions/runs/10910791157
* https://github.com/mozilla-mobile/firefox-ios/actions/runs/10911679329 

After this PR is merged, their corresponding Jenkins jobs for the `main` branch will be disabled.
* focus-iphone-iOS-15
* focus-iphone-iOS-16
* focus-ios-smoke-test-ipad-M1
* focus-ios-smoke-test-iphone-M1
* firefox-ios-smoketest-ipad
* firefox-ios-smoketest-iphone

Firefox full functional tests are not ready for Github Actions at the moment.     

## :pencil: Checklist
You have to check all boxes before merging
- [X] Filled in the above information (tickets numbers and description of your work)
- [ ] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: .github/workflows/firefox-ios-ui-tests.yml
Comment (isabelrios) [inline .github/workflows/firefox-ios-ui-tests.yml:193]: Just to double check, this workflow is only for the Smoketest?

Comment (clarmso) [reply .github/workflows/firefox-ios-ui-tests.yml:193]: Yes, I encountered plenty of timeouts on the full functional tests.

**Comment by clarmso** [comment]
Sorry. The webhook is not working at the moment. Let me resolve the issue before removing the "Do not merge" label.
https://github.com/mozilla-mobile/firefox-ios/actions/runs/10907058240/job/30270042275#step:10:27

**Comment by clarmso** [comment]
> Sorry. The webhook is not working at the moment. Let me resolve the issue before removing the "Do not merge" label. https://github.com/mozilla-mobile/firefox-ios/actions/runs/10907058240/job/30270042275#step:10:27

The updated webhook works now. I have removed the "Do not merge" label.

**Review by isabelrios — APPROVED** [review]

**Pull request merged**
